vscode-smart-contracts/package.json

```diff
@@ -23,6 +23,50 @@
       {
         "command": "concordium-smart-contracts.version",
         "title": "Concordium Smart Contracts: cargo-concordium version"
+      },
+      {
+        "command": "concordium-smart-contracts.build",
+        "title": "Concordium Smart Contracts: Build contract"
+      },
+      {
+        "command": "concordium-smart-contracts.build-skip-schema",
+        "title": "Concordium Smart Contracts: Build contract (no schema)"
+      },
+      {
+        "command": "concordium-smart-contracts.build-embed-schema",
+        "title": "Concordium Smart Contracts: Build contract (embed schema)"
```

Comment:
I would say that at least from a front-end development perspective, it is still the `base64` schema that is easiest (most convenient) to use currently with our tooling. I have just one schema variable (with the `base64` schema) at the front end that can be fed into every `serialization/invoke/sendTransaction` helper function or the browser wallet and it sorts out for me which schema I need from the blob. 

For example, while still in the active smart contract development phase where the schema changes often, it is much easier to just update one `base64` variable at the front-end, while `json` schema might need more adjustments.

While `json` schema, I think is the better way to represent the schema because it is human-readable (and I would prefer to have the `json` schema completely decoded in a human-readable manner). Currently, it is a bit annoying that our tooling needs to get exactly fed in which part of the `json` schema blob it should use. So my suggestion is actually more unrelated to this pull request but update our front-end tooling that it get the whole `json` schema blob fed into the `serialization/invoke/sendTransaction/browserWallet` and then our tooling sort out based on the entrypoint name which of the schema in the blob is appropriate to use.